### PR TITLE
Improve error message about missing `websockets` module

### PR DESCRIPTION
**Pull request by tomplus**
A tiny fix to help copy&paste a pip command to install websockets.
Thanks.

### CODE REVIEW

File: pylsp/python_lsp.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def start_ws_lang_server(port, check_parent_process, handler_class):
         import websockets
     except ImportError as e:
         raise ImportError(
-            "websocket modules missing. Please run pip install 'python-lsp-server[websockets]"
+            "websocket modules missing. Please run 'pip install python-lsp-server[websockets]'"
```
Comment (rchl) [inline pylsp/python_lsp.py]: It's the `python-lsp-server[websockets]` part that needs quotes to work in a shell.

Comment (tomplus) [reply pylsp/python_lsp.py]: Interesting, in my terminal it's not necessary, so I wanted to use quotes to mark a part to copy.
What about adding a colon and quotes as you suggest (it works for me):

```
websocket modules missing. Please run: pip install 'python-lsp-server[websockets]'
```

Comment (rchl) [reply pylsp/python_lsp.py]: sounds good to me

**Review by ccordoba12 — APPROVED** [review]
Thanks for your contribution @tomplus!

**Pull request merged**